R/0_generics.R

```diff
@@ -294,3 +294,29 @@ ff_starters <- function(conn, ...) {
 ff_starters.default <- function(conn, ...) {
   stop(glue::glue("No method of ff_starters found for platform: {conn$platform}."))
 }
+
+#' @export
+ff_playerscores.default <- function(conn, season, week, ...) {
```

Comment:
Hmm! Looks like this is an extra copy of the ff_playerscores default - it's also on line 187. Probably could just remove this whole thing